README.md

```diff
@@ -1 +1,38 @@
-# A-setup-and-mobile-version-skeleton
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
Please replace the text "Project Name" with your Project Title. For Example "Davis's Portfolio" or anything you like to name it 👍🏻